### PR TITLE
CPT-108: feat form validation

### PR DESCRIPTION
**Pull request by 634750802**

### CODE REVIEW

File: packages/ui/src/components/countDown/CountDown.component.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  useEffect(() => {
+    setTime(total);
+    const handle = setInterval(() => {
+      setTime((time) => {
+        if (time >= interval) {
+          return time - interval;
+        } else {
+          clearInterval(handle);
+          R.is(Function, onFinished) && onFinished();
```
Comment (hustKiwi) [inline packages/ui/src/components/countDown/CountDown.component.js:31]: The type check is probably redundant as the onFinished has been declared as `PropTypes.func`.

File: packages/ui/src/components/verifyCodeInput/verifyCodeInput.styled.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import { Button as AntButton } from 'antd';
+import * as colors from '../../colors';
+import * as mixins from '../../mixins';
+
+export const SendEmailButton = styled(AntButton)`
+  height: 22px !important;
+  line-height: 22px !important;
+  padding: 0 4px !important;
```
Comment (hustKiwi) [inline packages/ui/src/components/verifyCodeInput/verifyCodeInput.styled.js:9]: We have already overwritten the AntD buttons for different sizes and use cases like it's mentioned in the UI doc.
Please refer to https://tidb-community-ui-git-cpt-108-form-validation-tidb-community.vercel.app/?path=/story/ant-design-button--primary

Should we align with the same sizes but not introducing any new sizes?

Screen Shot 2021-04-30 at 7 10 45 PM

File: packages/ui/stories/2_components/countDown/countDown.stories.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  );
+};
+
+export const SearchCompany = Template.bind({});
+
+SearchCompany.args = {
+  total: 10000,
+  interval: 1000,
+  onFinished: () => alert('count down finished!')
+};
```
Comment (hustKiwi) [inline packages/ui/stories/2_components/countDown/countDown.stories.js:27]: 👍

File: packages/ui/stories/2_components/remoteSelect/remoteSelect.stories.js [hunk 4]
```diff
@@ -1,30 +1,52 @@
...
-    })));
+const fetchCompanies = ({ companyList, delay }) => (word) => {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve(
+        companyList
+          .filter(({ name }) => name.indexOf(word) !== -1)
+          .map(({ name }) => ({ label: name, value: name })));
+    }, delay);
+  });
```
Comment (hustKiwi) [inline packages/ui/stories/2_components/remoteSelect/remoteSelect.stories.js:18]: Thanks @634750802 for decoupling the story from the mock API 👍

File: packages/ui/stories/2_components/upload/upload.stories.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Upload } from '@tidb-community/ui';
+import { getTitle } from '../utils';
+import { UploadOutlined } from '@ant-design/icons';
```
Comment (hustKiwi) [inline packages/ui/stories/2_components/upload/upload.stories.js:4]: I'd like to separate internal imports and exporter imports.

```javascript
import React from 'react';
import { Upload } from '@tidb-community/ui';
import { UploadOutlined } from '@ant-design/icons';

import { getTitle } from '../utils';
```

File: src/pages/account/organization/new/form/fields/IncumbencyCertOption.component.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+import { useFormikContext } from 'formik';
+import { Upload } from '@tidb-community/ui';
+import { Form } from 'formik-antd';
+import data from '../form.data';
+import { api } from '@tidb-community/datasource';
+
+const { incumbencyCert } = data.form.verificationType;
+
+// uploadIncumbencyCert 不传参数代表删除文件
```
Comment (hustKiwi) [inline src/pages/account/organization/new/form/fields/IncumbencyCertOption.component.js:11]: Please use English comment as the repo is open-source.

**Review by hustKiwi — APPROVED** [review]

File: packages/ui/src/components/countDown/CountDown.component.js [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+          return 0;
+        }
+      });
+    }, interval);
```
[the remark this answers is not in the document]
Comment (634750802) [reply packages/ui/src/components/countDown/CountDown.component.js:35]: Maybe we will need an 100ms case.

File: packages/ui/src/components/verifyCodeInput/verifyCodeInput.styled.js (hunk 2 above)
Comment (634750802) [reply packages/ui/src/components/verifyCodeInput/verifyCodeInput.styled.js:9]: It is a special button that placed into a `Input` element, and it's default size will break it's parent style. We could discuss this at final.

File: src/pages/account/organization/new/form/fields/IncumbencyCertOption.component.js (hunk 6 above)
Comment (634750802) [reply src/pages/account/organization/new/form/fields/IncumbencyCertOption.component.js:11]: This comment is outdated and should be deleted.

**Pull request merged**